pages/jobs.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { s3 } from 'common/constants/urls';
+import ZipRecruiterJobs from 'components/ZipRecruiterJobs/ZipRecruiterJobs';
+import FeaturedJobItem from 'components/FeaturedJobItem/FeaturedJobItem';
+import Section from 'components/_common_/Section/Section';
+import FeaturedJobsData from 'components/FeaturedJobItem/featuredJobs.json';
+import styles from './styles/jobs.css';
```

Comment:
since u dropped the hero banner earlier, you can just delete the entire stylesheet and this import line